packages/fxa-settings/src/components/App/index.tsx

```diff
@@ -162,17 +165,40 @@ export const App = ({
   // Can be looked at in FXA-7626 or FXA-7184
   return (
     <Router basepath="/">
-      <AuthAndAccountSetupRoutes {...{ isSignedIn }} path="/*" />
-      <SettingsRoutes {...{ isSignedIn }} path="/settings/*" />
+      <AuthAndAccountSetupRoutes {...{ isSignedIn, integration }} path="/*" />
+      <SettingsRoutes {...{ isSignedIn, integration }} path="/settings/*" />
     </Router>
   );
 };
 
 const SettingsRoutes = ({
   isSignedIn,
-}: { isSignedIn: boolean } & RouteComponentProps) => {
+  integration,
+}: { isSignedIn: boolean; integration: Integration } & RouteComponentProps) => {
   const location = useLocation();
-  if (isSignedIn === false) {
+  // TODO: Remove this + config.sendFxAStatusOnSettings check once we confirm this works
+  const config = useConfig();
+
+  // Request and update account data/state to match the browser state for OAuth Sync flows
```

Comment:
@tarikeshaq LMK if there's any part of this comment that makes sense to tweak or add to!